v0/destinations/appcues/data/AppcuesPageProfile.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "destKey": "userId",
+    "sourceKeys": [
```

Comment:
use the sourceKey as listed in GenericFieldMappings as much as possible and use  getFieldValueFromMessage with that key.